core/commands/ipns.go

```diff
@@ -54,14 +54,6 @@ Resolve the value of another name:
 			return
 		}
 
-		if !n.OnlineMode() {
-			err := n.SetupOfflineRouting()
-			if err != nil {
-				res.SetError(err, cmds.ErrNormal)
-				return
-			}
-		}
-
```

Comment:
Because it is redundant with https://github.com/ipfs/go-ipfs/pull/1919/files#diff-153cb71153ee178fe2869148600ce8d5R222.